src/D2L.CodeStyle.Analyzers/ExtensionMethods.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Linq;
+using D2L.CodeStyle.Analyzers.Extensions;
+using Microsoft.CodeAnalysis;
+
+namespace D2L.CodeStyle.Analyzers {
+
+	internal static class ExtensionMethods {

```

Comment:
We have an extensions folder where we store our extension methods on roslyn types. There's a convention about how its organized by namespace -- could you move this into there?